.eslintrc

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Sounds bueno to me! I'd also be interested in adopting the js [standard](https://github.com/feross/standard) (no semicolons :scream_cat:) that way we wouldn't have to worry as much about maintaining it.